app/src/main/java/io/gnosis/safe/HeimdallApplication.kt

```diff
@@ -42,3 +49,26 @@ class HeimdallApplication : MultiDexApplication(), ComponentProvider {
         }
     }
 }
+
+private class CrashReportingTree : Timber.Tree() {
+
+    override fun log(priority: Int, tag: String?, message: String, t: Throwable?) {
+
+        if (priority == Log.VERBOSE || priority == Log.DEBUG) {
+            return
+        }
+        
+        t?.let {
+            if (priority == Log.ERROR) {
+                with(FirebaseCrashlytics.getInstance()) {
+                    setCustomKey(CRASHLYTICS_KEY_LOCALE, Locale.getDefault().language)
+                    recordException(it)
```

Comment:
If we log in this way, do we see in the Crashlytics dashboard that it was a non crashing exception?

Reply:
They should be reported as Non-fatal exceptions

Reply:
![image](https://user-images.githubusercontent.com/17750984/98924488-2c9c3200-24d5-11eb-816a-f1c581ab6923.png)
you can set this filter